neo/app/controllers/Application.scala

```diff
@@ -59,9 +59,11 @@ class Application @Inject()(val wsClient: WSClient,
     ActorFlow.actorRef(out => UserActor.props(out, cachesActor, USGeoGnosis))
   }
 
-  def tweet(id:String) = Action {
+  def tweet(id:String) = Action.async {
```

Comment:
I am new to Scala, and can give superficial comments: can we add comments to the code? :-)